src/signals/settings/routes.js

```diff
@@ -1,7 +1,7 @@
 const routes = {
```

Comment:
could we add a `export const baseUrl = `/instellingen/gebruikers`
and use this in the `routes` object

Then you could also import this base url when you want to push location to users instead of replacing with regex.

```js
import { baseUrl as usersRoute } from './routes'

history.push(`${usersRoute}/page/${page}`));
history.push(`${usersRoute}/${userId}`));
```

Reply:
Ah yes, I thought about doing that, but somehow I didn't get around to doing that. Will do.